kill.c

```diff
@@ -296,35 +296,64 @@ bool is_larger(const poll_loop_args_t* args, const procinfo_t* victim, procinfo_
             return false;
         }
         if (args->prefer_regex && regexec(args->prefer_regex, cur->name, (size_t)0, NULL, 0) == 0) {
-            cur->badness += BADNESS_PREFER;
+            if (args->sort_by_rss) {
+                return true;
+            } else {
+                cur->badness += BADNESS_PREFER;
+            }
         }
         if (args->avoid_regex && regexec(args->avoid_regex, cur->name, (size_t)0, NULL, 0) == 0) {
-            cur->badness += BADNESS_AVOID;
+            if (args->sort_by_rss) {
+                return false;
```

Comment:
This looks wrong

Reply:
If a process matches the avoid_regex, my only requirement is that it should not be terminated. I am not concerned about the amount of virtual memory (vm) it utilizes.

Reply:
got it.